server/web/routes/proposals.js

```diff
@@ -141,6 +148,14 @@ const register = function (server, options){
         Proposal.lookups
       );
 
+      for (const proposal of proposals.data) {
+        const feedbackCnt = await Feedback.count({
+          proposalId: proposal._id.toString(),
+          userId: user._id.toString(),
+        });
+        proposal.hasFeedback = feedbackCnt > 0;
+      }
+
```

Comment:
Need to change the logic here, since reviewers can probably only see their own feedback. but chair should be able to view everyone's feedback. This is a question for them to answer.